modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -238,13 +244,13 @@ public static real_t CubicInterpolateAngle(real_t from, real_t to, real_t pre, r
         public static real_t CubicInterpolateInTime(real_t from, real_t to, real_t pre, real_t post, real_t weight, real_t toT, real_t preT, real_t postT)
         {
             /* Barry-Goldman method */
-            real_t t = Lerp(0.0f, toT, weight);
-            real_t a1 = Lerp(pre, from, preT == 0 ? 0.0f : (t - preT) / -preT);
-            real_t a2 = Lerp(from, to, toT == 0 ? 0.5f : t / toT);
-            real_t a3 = Lerp(to, post, postT - toT == 0 ? 1.0f : (t - toT) / (postT - toT));
-            real_t b1 = Lerp(a1, a2, toT - preT == 0 ? 0.0f : (t - preT) / (toT - preT));
-            real_t b2 = Lerp(a2, a3, postT == 0 ? 1.0f : t / postT);
-            return Lerp(b1, b2, toT == 0 ? 0.5f : t / toT);
+            real_t t = Lerp(0f, toT, weight);
+            real_t a1 = Lerp(pre, from, preT == 0f ? 0f : (t - preT) / -preT);
+            real_t a2 = Lerp(from, to, toT == 0f ? 0.5f : t / toT);
```

Comment:
The preferred literals end in `.0f` not just `f`.

```suggestion
            real_t t = Lerp(0.0f, toT, weight);
            real_t a1 = Lerp(pre, from, preT == 0.0f ? 0.0f : (t - preT) / -preT);
            real_t a2 = Lerp(from, to, toT == 0.0f ? 0.5f : t / toT);
```

Reply:
First I have done the way you are suggesting, but I changed because I think is ugly have 0.0 - but would make more consistent, so I agree.